Formula/a/ampl-mp.rb

```diff
@@ -41,22 +41,21 @@ class AmplMp < Formula
     sha256 "1a4ef4cd1f4e8b959c20518f8f00994ef577e74e05824b2d1b241b1c3c1f84eb"
   end
 
-  def install
-    system "cmake", ".", *std_cmake_args, "-DBUILD_SHARED_LIBS=True"
-    system "make", "all"
-    if OS.mac?
-      MachO::Tools.change_install_name("bin/libasl.dylib", "@rpath/libmp.3.dylib",
-                                       "#{opt_lib}/libmp.dylib")
-    end
-    system "make", "install"
-
-    # Shared modules are installed in bin
-    mkdir_p libexec/"bin"
-    mv Dir[bin/"*.dll"], libexec/"bin"
+  # Install missing header files, remove in > 3.1.0
+  # https://github.com/ampl/mp/issues/110
+  patch do
+    url "https://github.com/ampl/mp/commit/8183be3e486d38d281e0c5a02a1ea4239695035e.patch?full_index=1"
+    sha256 "6b37201f1d0d6dba591e7e1b81fb16d2694d118605c92c422dcdaaedb463c367"
+  end
 
-    # Install missing header files, remove in > 3.1.0
-    # https://github.com/ampl/mp/issues/110
-    %w[errchk.h jac2dim.h obj_adj.h].each { |h| cp "src/asl/solvers/#{h}", include/"asl" }
+  def install
+    system "cmake", "-S", ".", "-B", "build",
+                    "-DAMPL_LIBRARY_DIR=#{libexec}/bin",
```

Comment:
Any reason why we're putting this there?

Reply:
No idea what these are. Just backwards compatible with what formula was manually moving. 

Default is `bin` and installs files with `.dll` extension which probably causes audit failures.

Reply:
I suggest moving it to `lib`. We can add a symlink if someone complains about breakage.

Reply:
We do have an audit to prevent non-standard library extensions like `.dll` (which should be Windows-specific but upstream decided to use it everywhere https://github.com/ampl/mp/blob/develop/CMakeLists.txt#L220).

Though it is only run on new formulae.